Managers/SettingsMan.cpp

```diff
@@ -469,7 +475,26 @@ namespace RTE {
 			}
 		}
 
-		writer << g_UInputMan;
+		writer.NewLine(false, 2);
```

Comment:
Ehhhh, I dunno about splitting. I sorta split them up into convenient categories in Save but breaking the methods into submethods following those categories introduces like 20 extra methods and at that point it's just bloat.
 
Maybe instead adding a reader line check to skip evaluation of blocks since the settings layout is never supposed to change.
It's not gonna make the gigantor methods any smaller, but it's at least gonna make them more efficient.

So like, if FrameMan properties are from line 1 to 10 and the reader is on line 11, you skip checking if `propName` is a FrameMan property entirely.
It can save a few nanoseconds because the further into the file it reads the more useless evaluation it needs to go through before hitting the right one (since the props are read in the same order they are written).

So something like
```
if (readerLine > 0 && readerLine <= 10) {
    if (propName == someFrameManProp) {
       blah
    } else if (propName == someFrameManProp) {
       blah
       etc etc etc
    }
} else if (readerLine > 10 && readerLine <= 20) {
    if (propName == someAudioManProp) {
       blah
    } else if (propName == someAudioManProp) {
       blah
       etc etc etc
    }
}
````

Reply:
Hmmm, I don't really care about that almost unnoticeable speed boost for something that's done just a couple times at startup. If it was run every update, then sure, but as-is, the femtogainz would genuinely not be noticeable, and it makes the code harder to read.
That's my main complaint here by the way, these things are lengthy as hell and as we add more properties, they'll just get worse. If I wanna add a new property, I have to scroll through to the right subsection for my manager, and then figure out the right ordering to match up with my manager. This is very dancode-y to me, very basic lua scripter - just plop everything in one big fat function look the other way.

All that said, I do get why you don't wanna have to put together 20 submethods, it's a bunch of busywork, so I don't care if it's not done for this PR, but I think at some point one or other of us will consider these things to be at their breaking point and spend the time splitting it up.
Also, I think it wouldn't be unreasonable if you have 3 managers in each submethod (e.g. `ReadPropertyFromFrameTimerOrPostProcessMan`) or something, just to save some time. It's probably a little silly, but it's still definitely more readable than these gigantor methods.